src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -1539,7 +1556,9 @@ bool Eltwise::EltwiseData::operator==(const EltwiseData &rhs) const noexcept {
            onednnAlgorithm == rhs.onednnAlgorithm &&
            alpha == rhs.alpha &&
            beta == rhs.beta &&
-           gamma == rhs.gamma;
+           gamma == rhs.gamma &&
+           detectNegative == rhs.detectNegative &&
+           detectPositive == rhs.detectPositive;
```

Comment:
I would propose to avoid introduction of new fields, especially because they slow down hash computation. 
We already have `alpha` and `beta` floating point attributes in Eltwise node. You can set alpha == 1.0f if detectNegative == true (same for beta and detectPositive);
Then computation formula will be the following: dstf = (srcf == INF) ? beta : (srcf == -INF) ? alpha : 0,0f.

Reply:
Done